benchmark/common/result.py

```diff
@@ -85,8 +96,15 @@ def f1(self) -> float:
     def f1(self, f1) -> None:
         self._f1 = f1
 
+    def _divide(self, a: float, b: float) -> float:
+        try:
+            return a / b
+        except ZeroDivisionError:
+            return 0
```

Comment:
There should be something like NA
The cases must be investigated.

Reply:
Yes, after added category to result, there are few cases that counted just 0.
So the problems occurred when calculating results, and I added exception for `ZeroDivisionError`.

Reply:
It should check whether 'a' is `0` too.
Then rate must be 1. 
There is a logic issue

Reply:
``` python
0 / 1  # Or any numbers are 0.0.. isn't it?
```
So the rate should be 0.0.

Reply:
0/0 = 1 otherweise a / b = 1 (when a ==b)

Reply:
It occurs error on python 3.8..

``` bash
>>> 0 / 0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ZeroDivisionError: division by zero
```

So I don't think it's not a issue.